content/post/quality-and-speed.md

```diff
@@ -0,0 +1,134 @@
+Right?
+
+<div style='clear:both'></div>
+<!--more-->
+
+## Is time the main requirement?
+
+Let me post here something that was made for my youngest daughter's birthday.
+
+<img src='/img/snowhite.jpg' />
+
+One was [done by my
+sister](https://www.instagram.com/p/CEfSjD_jkFS/?utm_source=ig_web_copy_link)
+and the other by my niece. It's easy to see that we have a piece of work that is more
+polished than the other one. The structure is similar but there are several
+quality differences that are easy to spot.
+
+Now, my sister did the best version of Snow White _much faster_ than my niece.
+We can argue that the comparison is not fair, after all my sister has _more
+experience and skills_ than my niece.
+
+But I can also argue that _time_ is not the main requirement for quality. I
+could say that my niece would not be able to achieve better quality even if she
+had one more week or month.
+
+## Is skill set the main requirement?
+
+Okay, now lets play the devil's advocate on the skill issue. If we look at the
+figure, we could say for example that the face is more detailed than the skirt.
+So it's not just _skill_. With more time my sister could have done a better job
+on the whole figure.
+
+On software engineering we deal with a lot of different modules. We may have
+ones that are more critical and need more attention. And we have other
+interesting challenges. We could actually need to build several figures, we
+could have several people working and need to coordinate them and make sure they
+are all aligned and using the same standards, etc.
+
+All this with a time constraint that wouldn't allow for a proper figure to be
+built.
+
+And lets not forget change requests. We can all relate with building a snow
+white's face that is top notch and then the customer asks for Little Red Hidding
+Hood.
+
+## Where's the silver bullet?
+
+Time does impact quality. If we gave a deadline of 2 minutes, maybe both my
+sister and my niece would present us with the same output. On the other hand I
+can also find examples of more skillful engineers taking longer on certain
+tasks.
+
+For example, lets consider a task to deploy a small script that does something
+simple. Engineer A could do:
+
+* Build a script
+* Put it live
+
+While Engineer B would do:
+
+* Create a
+  [RFC](https://engineering-management.space/post/rfc-driven-development/) with
+  the plan and get it peer reviewed
+* Come up with an incremental plan to release the task, bit by bit
+* Create a proper CI environment and builds the project for every pull request
+* Implement the feature with proper tests
+* Get it peer reviewed
+* Write some documentation
+* Write a
+  [postmortem](https://engineering-management.space/post/postmortem-culture/) if anything went wrong
+
+Maybe I'm exaggerating, but you get the picture. There are many things we can do
+to have more quality. The best 10x developer in the world would take more time
```

Comment:
don't understand this phrase `The best 10x developer in the world would take more time to deliver.`